Shared/Samples/Analyze network with subnetwork trace/AnalyzeNetworkWithSubnetworkTraceView.swift

```diff
@@ -152,7 +152,7 @@ struct AnalyzeNetworkWithSubnetworkTraceView: View {
                     if let selectedAttribute = selectedAttribute {
                         Spacer()
                         Text(selectedAttribute.name)
-                            .foregroundColor(.secondary)
+                            .foregroundStyle(Color.secondary)
```

Comment:
```suggestion
                            .foregroundStyle(.secondary)
```
Different from colors, primary and secondary are [`HierarchicalShapeStyle`](https://developer.apple.com/documentation/swiftui/hierarchicalshapestyle) so that their type information can be omitted.

Reply:
It threw a bunch of error without it and wouldn't compile, I'll give it a shot, it's possible that it was a phantom xcode issue.